### PR TITLE
PAF-177 Validation for date of birth bug fix

### PR DESCRIPTION
**Pull request by muthrajan**
## What?
Missing Validation  - What is your date of birth? Page - [PAF-177](https://collaboration.homeoffice.gov.uk/jira/browse/PAF-177)

## Why?
When the validation was triggered the message was missing

## How?
- Added validation for  'before', after1900Validator in about-you-dob  fields  in fields/index.js
- Validation messages added for  date,  before and after in validation.json
 
## Testing?
Testing passing  locally

### CODE REVIEW

File: apps/paf/fields/index.js [hunk 1]
```diff
@@ -1327,7 +1328,8 @@ module.exports = {
   },
   'about-you-dob': dateComponent('about-you-dob', {
     isPageHeading: true,
-    mixin: 'input-date'
+    mixin: 'input-date',
+    validate: ['required', 'before', after1900Validator]
```
Comment (Rhodine-orleans-lindsay) [inline apps/paf/fields/index.js:1332]: date of birth is not a mandatory field so the 'required' validation should be removed.

File: apps/paf/translations/src/en/validation.json [hunk 2]
```diff
@@ -271,6 +271,12 @@
   "how-do-you-know-the-person": {
     "maxlength": "You can't use more than {{maxlength}} characters for your answer"
   },
+  "about-you-dob": {
+    "date": "Please enter a valid date",
+    "required": "Enter your date of birth",
```
Comment (Rhodine-orleans-lindsay) [inline apps/paf/translations/src/en/validation.json:276]: Remove the required message as the date of birth is not mandatory.

**Review by Rhodine-orleans-lindsay — CHANGES_REQUESTED** [review]
See comments (don't forget to amend your commit to match). Also, in the 'Why' section of your description rather than saying 'part of PAF requirement ' you can be more specific and say that when the validation was triggered, the message was missing (as it says in the jira ticket).

File: apps/paf/fields/index.js (hunk 1 above)
Comment (muthrajan) [reply apps/paf/fields/index.js]: "required' validation removed.

File: apps/paf/translations/src/en/validation.json (hunk 2 above)
Comment (muthrajan) [reply apps/paf/translations/src/en/validation.json]: "required' message removed.

**Review by Rhodine-orleans-lindsay — APPROVED** [review]
lgtm

**Pull request merged**